parts/k8s/windowscsiproxyfunc.ps1

```diff
@@ -12,7 +12,7 @@ function New-CsiProxyService {
     DownloadFileOverHttp -Url $CsiProxyPackageUrl -DestinationPath $binaryPackage
 
     tar -xzf $binaryPackage -C $tempdir
-    cp "$tempdir\build\server.exe" "$KubeDir\csi-proxy-server.exe"
```

Comment:
We may use following 2 copy commands to make it compatible:
```
cp "$tempdir\build\server.exe" "$KubeDir\csi-proxy.exe"
cp "$tempdir\build\csi-proxy.exe" "$KubeDir\csi-proxy.exe"
```

Reply:
Since alpha release is the first release for CSI proxy and it has csi-proxy.exe, I don't think we have to be backward compatible here.

Reply:
https://kubernetesartifacts.azureedge.net/csi-proxy/master/binaries/csi-proxy.tar.gz is already updated, and has the `csi-proxy.exe`, it already does not work with current aks-engine v0.48.0

Reply:
we may not need code level compatibility, @ZeroMagic will make a new csi-proxy.tar.gz  which contains following two binaries as a workaround, then both aks-engine master and v0.48.0 could work:
```
$tempdir\build\server.exe
$tempdir\build\csi-proxy.exe
```
In the future, we won't use master branch csi-proxy binary, we should always use published releases, e.g. https://kubernetesartifacts.azureedge.net/csi-proxy/v0.1.0/binaries/csi-proxy.tar.gz

Reply:
cc @chewong current azure disk csi driver on windows won't work due to this issue, @ZeroMagic will prepare a new csi-proxy.tar.gz as a workaround

Reply:
https://k8scsi.blob.core.windows.net/csi-proxy/v0.1.0/binaries/csi-proxy.tar.gz works on aks-engine master and v0.48.0

Reply:
Thanks for the fix!